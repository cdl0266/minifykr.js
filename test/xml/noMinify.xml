<krpano>
  <!-- Comment that should not be deleted -->
  <include url="noMinify2.xml" />

  <plugin name="noMinify" />

</krpano>