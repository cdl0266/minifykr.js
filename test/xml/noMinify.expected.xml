<krpano>
  <!-- Comment that should not be deleted -->
  <plugin name="nominify2" />
  <!-- Comment that should not be deleted in minify2 -->

  <plugin name="noMinify" />

</krpano>